lib/google/ads/google_ads/interceptors/logging_interceptor.rb

```diff
@@ -58,6 +58,13 @@ def request_response(request:, call:, method:, metadata: {})
               if response.respond_to?(:partial_failure_error) && response.partial_failure_error
                 @logger.debug { build_partial_failure_message(response) }
               end
+              @logger.debug {
```

Comment:
I think this should actually be built into the "build_summary_message" call above. I had to do some extra stuff for streaming requests because the request ID isn't available until the request is over, but for non-streaming requests I think the request ID is available as soon as we get the response.

Reply:
Moved the logic to inside build_summary_message